core/components/footer/footer.tsx

```diff
@@ -70,29 +87,12 @@ export const Footer = async () => {
   ];
 
   return (
-    <ComponentsFooter
-      contactInformation={data.settings?.contact ?? undefined}
-      copyright={
-        data.settings
-          ? `© ${new Date().getFullYear()} ${data.settings.storeName} – Powered by BigCommerce`
-          : undefined
-      }
-      logo={data.settings ? logoTransformer(data.settings) : undefined}
-      paymentIcons={[
-        <AmazonIcon key="amazon" />,
-        <AmericanExpressIcon key="americanExpress" />,
-        <ApplePayIcon key="apple" />,
-        <MastercardIcon key="mastercard" />,
-        <PayPalIcon key="paypal" />,
-        <VisaIcon key="visa" />,
-      ]}
+    <FooterSection
+      copyright={copyright}
+      logo={logo}
+      paymentIcons={paymentIconsArray}
       sections={sections}
-      socialMediaLinks={data.settings?.socialMediaLinks
-        .filter((socialMediaLink) => Boolean(socialIcons[socialMediaLink.name]))
-        .map((socialMediaLink) => ({
-          href: socialMediaLink.url,
-          icon: socialIcons[socialMediaLink.name]?.icon,
-        }))}
```

Comment:
Looks like the API has data on the social media links that should be available but we're completely ignoring that. We should preserve that functionality